src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -85,12 +85,14 @@ const List = ({fungibilityFilter, isSearching, canAddAmount}: ListProps) => {
 const NftList = () => {
   const wallet = useSelectedWallet()
   const navigation = useNavigation<TxHistoryRouteNavigation>()
-  const {tokenSelectedChanged, amountChanged} = useSend()
+  const {tokenSelectedChanged, amountChanged, targets, selectedTargetIndex} = useSend()
   const {closeSearch} = useSearch()
   const balances = useBalances(wallet)
 
   const {nfts} = useNfts(wallet)
-  const sortedNfts = nfts.sort((NftA, NftB) => sortNfts(NftA.name, NftB.name))
+  const amountsSelected = Object.keys(targets[selectedTargetIndex].entry.amounts)
```

Comment:
nothing to update here just a remark, iterating over `amount` using `Object.*` functions will be discouraged in the future, it should leverage `Amounts` helper, -> `Amounts.toArray(....amounts).map({tokenId} => tokenId)` ok for now.